sdk/core/azure-core/test/ut/bodystream_test.cpp

```diff
@@ -68,9 +68,9 @@ TEST(BodyStream, BadInput)
 {
   TestBodyStream tb;
   ASSERT_DEATH(tb.Read(NULL, 1), "");
-  ASSERT_DEATH(tb.Read(NULL, 1, Azure::Core::Context::ApplicationContext), "");
+  ASSERT_DEATH(tb.Read(NULL, 1, Azure::Core::Context{}), "");
```

Comment:
One of the motivations for this change was to enable running all azure-core tests locally in VS test explorer to pass.
With these test fixes, have we achieved that outcome?

Reply:
No, because the HTTP tests still fail because there is some concurrency problem with the CURL transport and the HTTP patch verb.